mitodl/micromasters
# Removed TOS checkbox and validation from signup page

#### What are the relevant tickets?

Closes #928 

#### What's this PR do?

Removed the TOS checkbox from the signup flow

#### How should this be manually tested?

Set `filled_out` to False for your user and go through the signup flow. Make sure you don't see any errors related to TOS.

#### Any background context you want to provide?

The nationality bit (part of the issue description) was taken care of in a different issue
